app-tasks/dags/aoi-monitoring/update_aoi_projects.py

```diff
@@ -0,0 +1,99 @@
+import datetime
+import logging
+import os
+
+from airflow.models import DAG
+from airflow.operators import PythonOperator, BranchPythonOperator
+
+from rf.utils.exception_reporting import wrap_rollbar
+
+rf_logger = logging.getLogger('rf')
+ch = logging.StreamHandler()
+ch.setLevel(logging.INFO)
+formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+ch.setFormatter(formatter)
+rf_logger.addHandler(ch)
+
+logger = logging.getLogger(__name__)
+
+default_args = {
+    'owner': 'raster-foundry',
+    'start_date': datetime.datetime(2017, 4, 1)
+}
+
+dag = DAG(
+    dag_id='update_aoi_projects',
+    default_args=default_args,
+    schedule_interval=None,
+    concurrency=os.getenv('AIRFLOW_DAG_CONCURRENCY', 24)
+)
+
+
+def check_whether_update_available(**context):
+    # check whether a project needs to update
```

Comment:
I feel like we only need one function here: `update_project_with_aois` -- that checks AOIs for a project and updates them if necessary.

What's the use of splitting out the tasks?